lib/coinbase/wallet.rb

```diff
@@ -363,10 +364,8 @@ def export
       Data.new(wallet_id: id, seed: @master.seed_hex)
     end
 
-    def faucet
-      Coinbase.call_api do
-        Coinbase::FaucetTransaction.new(addresses_api.request_faucet_funds(id, default_address.id))
-      end
+    def faucet(asset_id: nil)
+      default_address.faucet(asset_id: asset_id)
```

Comment:
Please add to the `def_delegators`:

https://github.com/coinbase/coinbase-sdk-ruby/blob/6dffeaa1a2de84b982644bd5d233d8244ec613e7/lib/coinbase/wallet.rb#L220-L229

This ensures that there is only one place to coordinate function parameters